docroot/modules/custom/va_gov_user/src/Service/UserPermsService.php

```diff
@@ -173,16 +173,18 @@ public function entitySections($tid) {
   /**
    * Checks to see if the current user has an admin level role.
    *
+   * @param bool $administrator_only
+   *   If check should only be for administrator role.
+   *
    * @return bool
    *   TRUE if has an admin role, FALSE otherwise.
    */
-  public function hasAdminRole() : bool {
+  public function hasAdminRole(bool $administrator_only = FALSE) : bool {
     $current_user_roles = $this->currentUser->getRoles();
     // Roles to consider as admin.
-    $admin_roles = [
-      'administrator',
-      'content_admin',
-    ];
+    $admin_roles = $administrator_only
+    ? ['administrator']
+    : ['administrator', 'content_admin'];
```

Comment:
Minor: Did a linter recommend this format?   I am not a fan of the three lines, makes it hard to read.
```suggestion
    $admin_roles = $administrator_only ? ['administrator'] : ['administrator', 'content_admin'];
```